extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -29,7 +29,7 @@ public final class TransactionManagerConfiguration {
      * If the value is not absolute then the directory is relative
      * to the <em>user.dir</em> system property.
      */
-    @ConfigItem(defaultValue = "ObjectStore")
+    @ConfigItem(defaultValue = "target/ObjectStore")
```

Comment:
TBH, I'm not exactly sure how to solve this problem. `target` is Maven specific and related to the build. It makes no sense to use if for let's say a Gradle build or even running in production mode.

@aloubyansky are you aware of somewhere we had to solve the same issue? Or do you have an idea on where we could store this without it being an issue, neither for the build of projects nor for running projects in production?